gamestonk_terminal/cryptocurrency/defi/smartstake_model.py

```diff
@@ -0,0 +1,47 @@
+"""SmartStake Model"""
+__docformat__ = "numpy"
+
+from typing import Union, Dict
+import requests
+import pandas as pd
+
+
+def get_luna_supply_stats(supply_type: str, days: int) -> pd.DataFrame:
+    """Get supply history of the Terra ecosystem
+
+    Source: [Smartstake.io]
+
+    Parameters
+    ----------
+    days: int
+        Day count to fetch data
+    supply_type: str
+        Supply type to unpack json
+
+    Returns
+    -------
+    pd.DataFrame
+        Dataframe of supply history data
+    """
+
+    payload: Dict[str, Union[int, str]] = {"type": "history", "dayCount": days}
+
+    response = requests.get(
+        "https://prod.smartstakeapi.com/"
+        "listData?key=2mwTEDr9zXJH323M&token=1643550425&app=TERRA",
```

Comment:
Hi @minhhoang1023 I am a little curious about what is this key and token from line 31: `key=2mwTEDr9zXJH323M&token=1643550425` ? Is this your personal one api key or something :D ? I would consider to move it into config_terminal.py

Reply:
@JakubPluta so this is the public key that I got from the Network tab in Inspect. I don't think they allow people to register for keys / tokens at the moment. that's why I hard-coded in the code itself - as there's not currently a way for users to register for one.

Reply:
Sure, thank you for explanation.